### PR TITLE
CIV-13988 Include call charge url for welsh

### PR DESCRIPTION
**Pull request by laarmada**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-13988

### Change description ###

include call charge url for welsh

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/views/external-urls.njk [hunk 1]
```diff
@@ -13,7 +13,12 @@
 {% set essentialCookiesUrl = 'https://hmcts-access.service.gov.uk/cookies' %}
 {% set enforceJudgementUrl = 'https://www.gov.uk/make-court-claim-for-money/enforce-a-judgment' %}
 {% set claimCreationSmartSurveyUrl = 'https://www.smartsurvey.co.uk/s/CMC_ExitSurvey_Claimant/?pageUrl=/claim/claimId/confirmation' %}
-{% set callChargesUrl = 'https://www.gov.uk/call-charges' %}
+{% if getLanguage() =='cy' %}
+  {% set callChargesUrl = 'https://www.gov.uk/costau-galwadau' %}
+{% else %}
+  {% set callChargesUrl = 'https://www.gov.uk/call-charges' %}
+{% endif %}
+{% set callChargesWelshUrl = 'https://www.gov.uk/costau-galwadau' %}
```
Comment (m-meulendijks-v1) [inline src/main/views/external-urls.njk:21]: Is this one (callChargesWelshUrl) still being used?

**Review by m-meulendijks-v1 — APPROVED** [review]

File: src/main/views/external-urls.njk (hunk 1 above)
Comment (laarmada) [reply src/main/views/external-urls.njk:21]: yes it´s used across all nunjuck, we are using at this way:
image

Comment (laarmada) [reply src/main/views/external-urls.njk:21]: good point, let me remove it.

Comment (m-meulendijks-v1) [reply src/main/views/external-urls.njk]: Resolved now.

**Review by dtortolaV1 — APPROVED** [review]

**Pull request merged**